### ISSUE TITLE
How i can initial Content editorState ?

### ISSUE THREAD

**Issue by kenshero** [brijeshb42/medium-draft#92]
I save content editorState to database. And i want get content editorState to Editor component (edit content  feature) . I get content from my api. and how i update editorState ?  

```
componentDidMount {
  const content = getAPI()
  this.setState({
   editorState: content,
  })
}

render () {
        <Editor
            ref="editor"
            editorState={this.state.editorState}
            onChange={this.onChangeContent}
          />
}
```

Can you help me ?

**Comment by brijeshb42** [comment]
The demo editor uses a kind of api logic to fetch and set data. Checkout example.js

**Comment by kenshero** [comment]
It not working .

it show on log 
`convertFromRawToDraftState.js?8ef075a:42 Uncaught (in promise) TypeError: Cannot convert undefined or null to object`


my code for convert.

```
const nextState = {
      editorState: createEditorState(" some content xxxxx")
}

```

**Comment by brijeshb42** [comment]
A json object is supposed to be passed to `createEditorState`, not a string.

**Comment by brijeshb42** [comment]
Fixed in 0.5.4
